backend/routes/pixel.go

```diff
@@ -57,60 +65,108 @@ func getPixelInfo(w http.ResponseWriter, r *http.Request) {
 }
 
 func placePixelDevnet(w http.ResponseWriter, r *http.Request) {
+    reqBody, err := io.ReadAll(r.Body)
```

Comment:
This should be after the production check

Reply:
Oh, I understand. Noted